web/views.py

```
@@ -103,6 +105,9 @@
 )
 from .models import (
     Achievement,
+    AIChatMessage,
+    AIChatSession,
+    Avatar,
```

Ruff (0.8.2)

`.models.Avatar` imported but unused

Remove unused import

(F401)